utils/lib/grid_mapping.sql

```diff
@@ -0,0 +1,8 @@
+-- Grid location map
+CREATE TABLE IF NOT EXISTS grid_loc_map(
+    phy_tile_pkey INT,
+    vpr_tile_pkey INT,
+    FOREIGN KEY(phy_tile_pkey) REFERENCES phy_tile(pkey),
+    FOREIGN KEY(vpr_tile_pkey) REFERENCES tile(pkey)
+);
```

Comment:
Why not just have this in utils/lib/connection_database.sql?

Reply:
At first I thought there will be more tables related to the grid map. But yest it can be moved to the `utils/lib/connection_database.sql` file.